docs/book/src/topics/multitenancy.md

```diff
@@ -190,3 +196,4 @@ spec:
 ```
 
 For more details on how aad-pod-identity works, please check the guide [here](https://azure.github.io/aad-pod-identity/docs/).
```

Comment:
```suggestion
For more details on how workload identity works, please check the guide [here](https://azure.github.io/azure-workload-identity/docs/).
```

Reply:
or just remove this line if it's redundant information

Reply:
Line 158 should also be changed from 

```
The rest of the configuration is the same as that of service principal identity. This useful in scenarios where you don't want to have a dependency on [aad-pod-identity](https://azure.github.io/aad-pod-identity).
```

to 

```
The rest of the configuration is the same as that of the service principal identity. This is useful in scenarios where you don't want to have a dependency on [workload identity](https://azure.github.io/azure-workload-identity).
```

Reply:
I think, Manual service principal should not have a dependency on either workload identity or aad pod identity and infact it does not have any dependency.

Reply:
So removing that line makes sense.